index.js

```diff
@@ -39,7 +39,7 @@ module.exports = opts => {
 			}
 		}
 
-		childProcess.execFile(process.execPath, args, (err, stdout, stderr) => {
+		childProcess.execFile(process.execPath, args, {maxBuffer: 1024000}, (err, stdout, stderr) => {
```

Comment:
Make the number a constant like in https://github.com/sindresorhus/execa/blob/9cd5e2a39105fe3a366a0dc83ccac4ed2146f585/index.js#L14 for readability reasons.